packages/relay/src/lib/poller.ts

```diff
@@ -0,0 +1,132 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import { Eth } from '../index';
+import { Logger } from 'pino';
+
+export interface Poll {
+    tag: string,
+    callback: Function,
+    lastPolled?: string
+}
+
+const LOGGER_PREFIX = 'Poller:';
```

Comment:
nit: I think we need to add a requestId, like the rest of the relay. But this might be done at a later stage ?

Reply:
There is no request in this case. The Poller constantly checks for new data and returns it to `SubscriptionController`, which in turn sends that data to all subscribed clients. The data sent to the clients is actually not in the JSON RPC format - it doesn't have the `id` and `version` fields.